test/test_fx.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from torch.fx import GraphModule
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+class TestFX(TestCase):
```

Comment:
can you add test with passing dictionary explicitly as the argument, indexing it and constructing a dictionary in graph module (the most common cases)

Reply:
Added a test that exercises what I think you're describing, please take a look